src/test/java/uk/gov/companieshouse/api/strikeoffobjections/model/chips/ChipsRequestTest.java

```diff
@@ -102,4 +107,28 @@ void testToStringNullAttachments() {
 
         assertEquals(expectedOutput, actualOutput);
     }
+
+    @Test
```

Comment:
This unit test seems to be testing Jackson code/behaviour and not ours. Is it needed?

Reply:
good spot, i left didn't need the extra line in the test. Removed now. The test will pass if the ChipsRequest has the @JsonInclude(JsonInclude.Include.NON_NULL)
annotation and fail if someone removes it.